cmake/Modules/FindComputeCpp.cmake

```diff
@@ -47,22 +47,19 @@ find_package(OpenCL REQUIRED)
 
 # Find ComputeCpp package
 
-if(DEFINED ComputeCpp_DIR)
```

Comment:
This is a behaviour change and I don't understand it. (Apart from anything else, you're going to break my builds.)

What's the intent? I believe it also is counter to the CMake guidelines for how to write Find files.

Reply:
Why would this break your build? What's the use case?

The new behaviour should be the same as the old one, unless I'm missing something:
* If neither `ComputeCpp_DIR` nor `COMPUTECPP_DIR` env variable are defined, `ComputeCpp_DIR ` is set to empty, and the CMake configuration fails.
* The user can specify either variable, and it will be picked up by CMake. The CMake variable will have priority over the environment one.

Do CMake guidelines forbid options and cache variables in Find files? In that case we probably should drop this, yes.

Reply:
I see I'd slightly misread the second part of this change. Nevertheless AIUI the xxxx_DIR variable should not be cached so that the user can specify a different one in configuration.